CONTRIBUTING.md

```diff
@@ -1,187 +1,148 @@
 # Contributing
 
-FV3core is being actively developed by Vulcan Inc. Please contact us if there is interest
-in making contributions. Credit will be given to contributors by adding their names
-to the `CONTRIBUTORS.rst` file.
-
+FV3core is actively developed by Vulcan, so please contact us if there is interest in making contributions in the near-term.
+Contributors names will be added to [AUTHORS.rst](https://github.com/VulcanClimateModeling/fv3core/blob/master/AUTHORS.rst).
 
 ## Linting
 
-The linter may change over time, but you should always be able to correct your code (if you have pip installed requirements_lint.txt locally) using `make lint`, or to correct all files, including those not yet in version control `make lint_all`
```

Comment:
what is wrong with this sentence? this makes it clear that the linting requirements may change

Reply:
Literally anything can change in code, and as we know it is in fact always evolving. This document should describe the current state of the code and it is implied that change can happen.